### PR TITLE
Fix tutorial redirects

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Following up on #156 

### PR Checklist

- [ ] Tests are added
- [ ] Documentation, if applicable

### CODE REVIEW

**Comment by ToucheSir** [comment]
I just checked all the links on http://fluxml.ai/tutorials/ and none of them are broken. Is there a reason for this specific change?

**Comment by Saransh-cpp** [comment]
The link for each tutorial post went stale - https://fluxml.ai/tutorialposts/2020-09-15-deep-learning-flux/

File: tutorialposts/2021-10-14-vanilla-gan.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 title = "Generative Adversarial Networks"
 published = "14 October 2021"
 author = "Ralph Kube"
-external = "http://fluxml.ai/Flux.jl/stable/tutorials/2021-10-14-vanilla-gan/"
```
Comment (Saransh-cpp) [inline tutorialposts/2021-10-14-vanilla-gan.md:5]: `external` makes the link in `/tutorials` work (attaches an external link to the bullets)

File: tutorialposts/2021-10-08-dcgan-mnist.md [hunk 2]
```diff
@@ -2,5 +2,8 @@
 title = "Deep Convolutional Generative Adversarial Network (DCGAN)"
 published = "8 October 2021"
 author = "Deeptendu Santra"
-external = "http://fluxml.ai/Flux.jl/stable/tutorials/2021-10-08-dcgan-mnist/"
 +++
+
+~~~
+<meta http-equiv="refresh" content="0; url='http://fluxml.ai/Flux.jl/stable/tutorials/2021-10-08-dcgan-mnist/'"/>
+~~~
```
Comment (Saransh-cpp) [inline tutorialposts/2021-10-08-dcgan-mnist.md:9]: This actually makes each `/tutorialpost` link redirect to another link.

**Review by Saransh-cpp — COMMENTED** [review]
Maybe we should keep both `meta` and `external` in each file?

**Comment by ToucheSir** [comment]
Yes, both would be good then.

**Review by ToucheSir — APPROVED** [review]
Thanks!

**Pull request merged**